measures/ResStockArguments/measure.rb

```diff
@@ -599,7 +599,9 @@ def run(model, runner, user_arguments)
       # Error check MF & SFA geometry
       if !has_rear_units && ((corridor_position == 'Double-Loaded Interior') || (corridor_position == 'Double Exterior'))
         runner.registerWarning("Specified incompatible corridor; setting corridor position to 'Single Exterior (Front)'.")
-        corridor_position = 'Single Exterior (Front)'
+        # FIXME: This doesn't actually do anything because corridor_position is not used later.
+        # Remove code? Or should the code be moved up?
+        # corridor_position = 'Single Exterior (Front)'
```

Comment:
@aspeake @joseph-robertson Any idea how this code should be fixed?

Reply:
@shorowit I think the simplest fix would be to move the block above this line: https://github.com/NREL/resstock/blob/d2c21d59832b14560ef8da7da2e3059ff7aea69c/measures/ResStockArguments/measure.rb#L586

This scenario would fall under the `else` in the logic above, and should only apply to MF.